src/domain/DemoData.ts

```diff
@@ -21,16 +21,21 @@ export default class DemoData {
 
         let tx = new MariaTransaction(this.pool)
 
-        let lAddres = new Addres('Gardenstreet', '42c', '12345', 'Gardencity', 'Gardencountry')
+        let lAddress = new Address('Gardenstreet', '42c', '12345', 'Gardencity', 'Gardencountry')
 
         let luisa = new Knight({
             name: 'Luisa',
-            addres: lAddres
+            address: lAddress
         })
 
-        log.admin('luisa', luisa)
-        let result = await this.knightLogic.store(luisa, tx)
-        log.admin('created knight', result)
-
+        let alreadyExist = await this.knightLogic.read({name: 'Luisa'}, tx)
```

Comment:
Wie wäre es denn hier mit einer Funktion, in die ich eine beliebige Instanz reingeben kann. Die Funktion überprüft dann erstmal, ob die Instanz schon existiert und ruft dann bei Bedarf `store` auf.